munch/__init__.py

```diff
@@ -35,7 +35,10 @@
     # package is not installed
     pass
```

Comment:
I don't know if this is the desired behaviour. But if we want to default to version 0.0.0 if we cannot read the version, we could do this here.

```suggestion
    # package is not installed
    __version__ = "0.0.0"
```
Then the original code on line 38 could remain the same.

Reply:
nicely done! 
I think we should keep both your solution and @mdornseif , as @mdornseif fixes also cases where the version does not contain numbers